### PR TITLE
Update plugin with object info and change Pay.nl name to PAY.

### PR DESCRIPTION
**Pull request by max-geraci**

### CODE REVIEW

File: upload/admin/language/nl-nl/extension/payment/paynl_payconiq.php [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?php
+include 'paynl3.php';
+// Heading
+$_['heading_title'] = 'Pay.nl - Payconiq';
```
Comment (daan-pay) [inline upload/admin/language/nl-nl/extension/payment/paynl_payconiq.php]: PAY.

File: upload/admin/language/nl-nl/extension/payment/paynl_multibanco.php [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?php
+include 'paynl3.php';
+// Heading
+$_['heading_title'] = 'Pay.nl - Multibanco';
```
Comment (daan-pay) [inline upload/admin/language/nl-nl/extension/payment/paynl_multibanco.php]: PAY.

File: upload/admin/language/en-gb/extension/payment/paynl_payconiq.php [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<?php
+include 'paynl3.php';
+// Heading
+$_['heading_title'] = 'Pay.nl - Payconiq';
```
Comment (daan-pay) [inline upload/admin/language/en-gb/extension/payment/paynl_payconiq.php]: PAY.

File: upload/admin/language/en-gb/extension/payment/paynl_multibanco.php [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<?php
+include 'paynl3.php';
+// Heading
+$_['heading_title'] = 'Pay.nl - Multibanco';
```
Comment (daan-pay) [inline upload/admin/language/en-gb/extension/payment/paynl_multibanco.php]: PAY.

**Review by daan-pay — APPROVED** [review]

**Review by woutse — APPROVED** [review]

**Pull request merged**